lib/Server.js

```diff
@@ -580,17 +581,93 @@ class Server {
   }
 
   showStatus() {
-    const suffix = '/';
-    const uri = `${createDomain(this.options, this.server)}${suffix}`;
+    const useColor = getColorsOption(getCompilerConfigArray(this.compiler));
 
-    const configArr = getCompilerConfigArray(this.compiler);
-    const colors = getColorsOption(configArr);
+    const protocol = this.options.https ? 'https' : 'http';
+    const { hostname, port } = this;
+    const prettyPrintUrl = (newHostname) =>
+      url.format({ protocol, hostname: newHostname, port, pathname: '/' });
+
+    let prettyHostname;
+    let lanUrlForTerminal;
+
+    if (hostname === '0.0.0.0' || hostname === '::') {
+      prettyHostname = 'localhost';
+
+      const localIP = internalIp.v4.sync();
```

Comment:
```suggestion
      const localIP = hostname === '::' ? internalIp.v6.sync() : internalIp.v4.sync();
```

Reply:
It is for terminal output, developer should copy past it from terminal, IPv6 is not supported in many browsers

Reply:
`url` automatically formats IPv6 hostname with brackets, which most browsers support including IE 11, e.g., `http://[::]:8080`. I think IPv6 should take precendence over IPv4, if available.

Reply:
make sense, I hope this does not scare developers